src/components/ContextMenu.scss

```diff
@@ -46,6 +53,13 @@
   .context-menu-option:hover {
     color: var(--popup-background-color);
     background-color: var(--select-highlight-color);
+
+    &.dangerous {
+      div:nth-child(1) {
+        color: var(--popup-background-color);
```

Comment:
this color seems to be the same with it's parent.. let's remove it.

Reply:
```
div:nth-child(1) {
    color: var(--popup-background-color)
}
```
If I remove this rules then color of the text will remain red due to specificity.

Reply:
right :|